package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluid-publish",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
You'll need to update from master as a new change has already gone in and has bumped the version to 2.1.0. You'll probably want to set your version to 2.1.1.

Reply:
Sure, I will.